app/controllers/marine_controller.rb

```diff
@@ -145,21 +144,8 @@ def designations
     end
   end
 
-  def total_coverage
-    @totalMarineProtectedAreas = 15357
-    @oceanProtectedAreasPercent = 6.40
-    @oceanProtectedAreasKm = 23183060
-  end
-
-  def distributions
-    @distributions = {
-      nationalWaters: 39,
-      nationalWatersPa: 16.02,
-      nationalWatersKm: 22624944,
-      highSeas: 61,
-      highSeasPa: 0.25,
-      highSeasKm: 558116
-    }
+  def marine_statistics
+    @marine_statistics = Wdpa::MarineStatsImporter.stats
```

Comment:
I would probably retrieve the stats using a different module or just with a plain Redis call rather than using the importer. It can be confusing, also because "getting" stats doesn't sound like a function of an "importer"

Reply:
Great point @Levia! I will make an amendment.